server.js

```diff
@@ -35,6 +35,6 @@ app.use(express.static(path.join(__dirname, "public")));
 
 app.use(routes);
 
-sequelize.sync({ force: false }).then(() => {
+sequelize.sync({ force: true }).then(() => {
```

Comment:
Maybe best not to flush db on restart
```suggestion
sequelize.sync({ force: false }).then(() => {
```